core/capabilities/webapi/webapicap/webapi_target-schema.json

```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/smartcontractkit/chainlink/v2/core/capabilities/webapi/webapicap/web-api-target@1.0.0",
+    "description": "A target that sends HTTP requests to a URL",
+    "$defs": {
+        "TargetPayload": {
+            "type": "object",
+            "properties": {
+                "url": {
+                    "type": "string",
+                    "description": "The URL to send the request to"
+                },
+                "method": {
+                    "type": "string",
+                    "description": "The HTTP method to use for the request"
+                },
+                "headers": {
+                    "type": "object",
+                    "description": "The headers to include in the request",
+                    "additionalProperties" : {
+                        "type": "string"
+                    }
+                },
+                "body": {
+                    "type": "string",
+                    "description": "The body of the request"
+                }
+            },
+            "required": ["url"],
+            "additionalProperties": false
+        },
```

Comment:
Does this need to be a signed report, or is it ok not to for now because we're waiting for F+1 nodes?  Long-term, I think this should receive a report and be configured to either send that to the user or to extract the body first.  Obviously, many Web 2 systems won't be able to take a report, but capable users should be able to verify the signatures if they want to.

Reply:
I don't follow, all nodes will send data to the capability, right?  Does it do deduplication, or will the user get up to 3F+1 copies?  For now, it's ok, but later we'll need to solve this.